src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,32 @@
+import { Link } from "react-router-dom";
+
+export const Navbar = ({ drawerDisplay, setDrawerDisplay }) => {
+  return (
+    <nav className="nav">
+      <div className="nav-section">
+        <button
+          className="material-icons nav-icon header-navigation-icon"
+          onClick={() =>
+            drawerDisplay === "flex"
```

Comment:
The drawer doesn't close when I click on it for the first time. I guess it's because the state of drawerDisplay is set to "" initially. So when I click on the menu button for the first time this condition returns false & on the first click sets the display to "flex" & then when I click on the menu button for the second time, that is when the state gets the value "none". I just thought of pointing it out for you, if it wasn't the desired behavior.